### PR TITLE
Add Xarray suggestions

### PR DESCRIPTION
**Pull request by dcherian**
This is looking good. I like it!

### CODE REVIEW

File: notebooks/04-dask-xarray.ipynb [hunk 1]
```diff
@@ -4961,8 +4974,23 @@
...
+    "https://tutorial.xarray.dev/intermediate/xarray_and_dask.html#getting-concrete-values"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8ce5b8a3-cea9-4d04-bac0-dd574da5877d",
+   "metadata": {
+    "jp-MarkdownHeadingCollapsed": true,
+    "tags": []
+   },
```
Comment (dcherian) [inline notebooks/04-dask-xarray.ipynb:4993]:
```suggestion
```

File: notebooks/04-dask-xarray.ipynb [hunk 2]
```diff
@@ -4961,8 +4974,23 @@
...
+    "For convenience Xarray also provides `.compute` and `.persist`. These will return an Xarray object:\n",
+    "\n",
+    "https://tutorial.xarray.dev/intermediate/xarray_and_dask.html#getting-concrete-values"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8ce5b8a3-cea9-4d04-bac0-dd574da5877d",
+   "metadata": {
+    "jp-MarkdownHeadingCollapsed": true,
```
Comment (dcherian) [inline notebooks/04-dask-xarray.ipynb:4991]:
```suggestion
```

File: notebooks/04-dask-xarray.ipynb [hunk 3]
```diff
@@ -4961,8 +4974,23 @@
   },
   {
    "cell_type": "markdown",
-   "id": "8ce5b8a3-cea9-4d04-bac0-dd574da5877d",
+   "id": "889e9971-43eb-4837-8d13-e560de21f908",
    "metadata": {},
+   "source": [
+    "### TODO: compute, persist, and load\n",
+    "\n",
+    "For convenience Xarray also provides `.compute` and `.persist`. These will return an Xarray object:\n",
```
Comment (dcherian) [inline notebooks/04-dask-xarray.ipynb:4982]: This is my only big suggestion. May want to cover the differences between these three. It seems to be a common question.

Comment (negin513) [reply notebooks/04-dask-xarray.ipynb:4982]: Thanks for pointing this out. Yup! I completely forgot about that. Will add a paragraph about them three of them.

**Comment by negin513** [comment]
Thanks Deepak! for reviewing this. I really appreciate it and I like the changes you made. I will add a sentence about their different in a PR later tonight.

**Pull request merged**